python/common/python/time.java

```diff
@@ -89,10 +89,30 @@ public time() {
     }
 
     @org.python.Method(
-            __doc__ = ""
+            __doc__ = "ctime(seconds) -> string\n\n"+
+                      "Convert a time in seconds since the Epoch to a string in local time.\n"+
+                      "This is equivalent to asctime(localtime(seconds)). When the time tuple is\n"+
+                      "not present, current time as returned by localtime() is used.",
+            default_args = {"seconds"}
     )
-    public static org.python.Object ctime() {
-        throw new org.python.exceptions.NotImplementedError("time.ctime() has not been implemented.");
+    public static org.python.Object ctime(org.python.Object seconds) {
+        java.util.Date date;
+        if(seconds == null){
+            long currentTimeInMillis = System.currentTimeMillis();
+            date = new java.util.Date(currentTimeInMillis);
+        } else {
+            date = new java.util.Date(((org.python.types.Int) seconds.__int__()).value * 1000L);
+        }
+        java.text.SimpleDateFormat ft = new java.text.SimpleDateFormat("E MMM dd HH:mm:ss yyyy");
+        String padded_date = ft.format(date);
+        if(Character.toString(padded_date.charAt(8)).equals("0")) {
+            java.lang.StringBuilder sb = new StringBuilder();
+            sb.append(padded_date.substring(0,7));
+            sb.append("  ");
+            sb.append(padded_date.substring(9,padded_date.length()));
+            padded_date = sb.toString();
+        }
+        return new org.python.types.Str(padded_date);
```

Comment:
Can you please run `ant checkstyle` locally and fix the style errors that it finds?
Normally we can see these in the CI, but [it seems to not be working correctly right now](https://github.com/pybee/voc/issues/640).

Reply:
Yeah sure.